### PR TITLE
Fix handling of COLRv0 layers

### PR DESCRIPTION
**Pull request by khaledhosny**
Glyphs handles components for color layers in a special way. If the component glyph has color layers of its own, the component use the first color layer with the same color index, otherwise it fallback to the default layer. We try to do that here as well.

Fixes https://github.com/googlefonts/glyphsLib/issues/866

### CODE REVIEW

**Comment by khaledhosny** [comment]
@yanone can you test this? I didn’t test with Cairo, but I tested with another font of mine that had a similar issue.

**Comment by yanone** [comment]
@khaledhosny Works for Cairo Play 👍 Thank you

**Comment by anthrotype** [comment]
Thanks Khaled!

File: Lib/glyphsLib/builder/components.py [hunk 1]
```diff
@@ -46,30 +46,36 @@ def to_ufo_components(self, ufo_glyph, layer):
...
+            ]
+            for i, l in enumerate(color_layers):
+                if l._color_palette_index() == layer._color_palette_index():
+                    if l.layerId != l.associatedMasterId:
+                        # If it is not a master layer, we rename it in
+                        # _to_ufo_color_palette_layers(), so we reference the
+                        # same name here.
+                        component_name += f".color{i}"
+                    break
+            print(ufo_glyph.name, component_name)
```
Comment (anthrotype) [inline Lib/glyphsLib/builder/components.py:70]: I presune this `print` was a leftover? I'll delete it

Comment (anthrotype) [reply Lib/glyphsLib/builder/components.py:70]: ah you already did remove it, thank you!
https://github.com/googlefonts/glyphsLib/commit/896817d12798f9bddc374392913c0ecd1dd3dce7

**Pull request merged**